### DISCUSSION TITLE
How to change icon favicon?

### DISCUSSION THREAD

**Discussion by seanchien4** [daattali/beautiful-jekyll#1051] [Q&A] [answered]
I looked for the Q&A and it says just add favicon.io into root dictionary, and I did it but I did not change anything for a while. 
Anyone has any idea how to change it? thanks

**Comment by daattali** [comment, marked as answer]
It takes some time. It does work, I see a little fish icon 
![image](https://user-images.githubusercontent.com/952340/196021894-7cde9a99-f0c9-4ee6-8273-043d2cbea742.png)

**Reply by seanchien4** [reply]
Yes I see it now. However, it only show on google browser. I know favicon files should have multiple formats to show on different browsers or on phones. Do you add multiple favicon files? If so, how do you name it? Thanks!

**Reply by daattali** [reply]
No, I see it on other browsers too. Andanyway this is not a beautifuljekyll issue it's a geenral web thing. But it does work.
